### PR TITLE
XT-1517 - Host ixviewer.js on the CDN

### PR DESCRIPTION
**Pull request by derekgengenbacher-wf**
#### Description:
- Host the ixviewer.js on the CDN for easy download

#### Testing:
Hit `https://cdn-dev.wdesk.org/ixbrl-viewer/<build_id>/ixbrlviewer.js` and verify that the ixviewer.js is hosted correctly on the CDN.

**review**:
@Workiva/xt

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: Dockerfile [hunk 1]
```diff
@@ -35,6 +35,11 @@ RUN npm run stylelint
 # Upload ixbrlviewer.js to github artifacts
 ARG BUILD_ARTIFACTS_GITHUB_RELEASE_ASSETS=/build/iXBRLViewerPlugin/viewer/dist/ixbrlviewer.js
 
+# Host ixviewer.js on CDN
+RUN mkdir /static_release
+RUN tar -czf /static_release/assets.tar.gz -C /build/iXBRLViewerPlugin/viewer/dist/ .
```
Comment (sagesmith-wf) [inline Dockerfile:40]: Nit: Should you tighten this up, or is it true that we want all of the files from the dir forever and not just the ixviewer.js

**Review by sagesmith-wf — APPROVED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (derekgengenbacher-wf) [reply Dockerfile:40]: There is only one file in `/build/iXBRLViewerPlugin/viewer/dist/` and that is the ixviewer.js.

**Comment by austinmatherne-wk** [comment]
### Verified

* That the js build artifact is hosted on the CDN: https://cdn-dev.wdesk.org/ixbrl-viewer/2983549/ixbrlviewer.js

QA +10
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**